src/substrait/textplan/tests/CMakeLists.txt

```
@@ -9,3 +9,15 @@ target_link_libraries(
         parse_result_matchers
         parse_result
         gmock)
+
+add_test_case(
+        symbol_table_test
+        SOURCES
+        SymbolTableTest.cpp
+        EXTRA_LINK_LIBS
+        symbol_table
+        substrait_proto
+        fmt::fmt-header-only
+        gmock
+        gtest
+        gtest_main)
```

Newline here to end the file.
```suggestion
        gtest_main)

```